modules/ipi-install-replacing-a-bare-metal-control-plane-node.adoc

```diff
@@ -0,0 +1,254 @@
+// This is included in the following assemblies:
+//
+// installing/installing_bare_metal_ipi/ipi-install-expanding-the-cluster.adoc
+
+:_content-type: PROCEDURE
+[id="replacing-a-bare-metal-control-plane-node_{context}"]
+= Replacing a bare-metal control plane node
+
+Use the following procedure to replace an installer-provisioned {product-title} control plane node.
+
+[IMPORTANT]
+====
+If you reuse the `BareMetalHost` objects from an existing control plane host, do not leave the `externallyProvisioned` field set to `true`.
+
+Existing control plane `BareMetalHost` objects may have the `externallyProvisioned` flag set to `true` if they were provisioned by {product-title}.
+====
+
+.Procedure
+
+. Remove the old `baremetalhost`, machine and node objects:
++
+[source,terminal]
+----
+$ oc delete node kni1-vmaster-0
+$ oc delete bmh -n openshift-machine-api vmaster-0
+$ oc delete machine -n openshift-machine-api kni1-master-0
+----
++
+. Create the new `baremetalhost` object and the secret to store the BMC credentials:
++
+[source,terminal]
+----
+$ cat <<EOF | oc apply -f -
```

Comment:
This is not ok. You need to create both the secret to store the BMC credentials AND the `baremetalhost` object:

```
$ cat <<EOF | oc apply -f -
apiVersion: v1
kind: Secret
metadata:
  name: ocp-master-2-bmc-secret
  namespace: openshift-machine-api
data:
  password: Y2Fsdmlu
  username: cm9vdA==
type: Opaque
---
apiVersion: metal3.io/v1alpha1
kind: BareMetalHost
metadata:
  name: ocp-master-2
  namespace: openshift-machine-api
spec:
  automatedCleaningMode: disabled
  bmc:
    address: redfish-virtualmedia+http://192.168.124.113:8000/redfish/v1/Systems/f87eaf82-b32d-4291-ace6-b28677964e78
    credentialsName: ocp-master-2-bmc-secret
  bootMACAddress: aa:aa:aa:aa:ab:03
  bootMode: UEFI
  externallyProvisioned: false
  hardwareProfile: unknown
  online: true
EOF
```